### PR TITLE
Add blueprint to Signed Distance Fields example

### PR DESCRIPTION
**Pull request by Wumpf**
### What

Also replace the somewhat broken screenshot it had (thumbnail was alright)

image

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using newly built examples: [app.rerun.io](https://app.rerun.io/pr/5635/index.html)
  * Using examples from latest `main` build: [app.rerun.io](https://app.rerun.io/pr/5635/index.html?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [app.rerun.io](https://app.rerun.io/pr/5635/index.html?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/5635)
- [Docs preview](https://rerun.io/preview/8bd47c05b89f3eefd1fda0e4a9564799b8c5a76d/docs) 
- [Examples preview](https://rerun.io/preview/8bd47c05b89f3eefd1fda0e4a9564799b8c5a76d/examples) 
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

### CODE REVIEW

File: src/mesh-to-sdf
Comment (teh-cmc) [inline src/mesh-to-sdf:1]: ?

**Review by teh-cmc — APPROVED** [review]
Looks great.

Looks like you inadvertently added a submodule though

File: examples/python/signed_distance_fields/README.md
Comment (teh-cmc) [inline examples/python/signed_distance_fields/README.md:1]: I cannot see any reason why this example isnt available in the app? We should add a `channel` here.

Comment (emilk) [reply examples/python/signed_distance_fields/README.md:1]: One reason is that is quite confusing. I think a README would be a pre-requisite for adding it to the app. Maybe also make it a bit prettier.

Comment (Wumpf) [reply examples/python/signed_distance_fields/README.md:1]: not doing that in this pr

**Comment by Wumpf** [comment]
> Looks like you inadvertently added a submodule though

urgh, thought I removed that again, wasn't thorough. The requirements.txt in this one does a clone which an innocent `git add .` will trip over then

**Pull request merged**